lib/binding_web/binding.c

```diff
@@ -184,6 +184,20 @@ TSTree *ts_parser_parse_wasm(
   return ts_parser_parse(self, old_tree, input);
 }
 
+/**********************/
+/* Section - Language */
+/**********************/
+
+int ts_node_type_is_named_wasm(const TSLanguage *self, TSSymbol typeId) {
+  const TSSymbolType symbolType = ts_language_symbol_type(self, typeId);
+  return symbolType == TSSymbolTypeRegular;
+}
+
+int ts_node_type_is_visible_wasm(const TSLanguage *self, TSSymbol typeId) {
```

Comment:
For consistency with the other functions, could you rename these new methods so that they begin with the prefix `ts_language_`?